mkosi/config.py

```diff
@@ -1382,13 +1386,15 @@ def parse_drive(value: str) -> Drive:
     if not is_valid_filename(id):
         die(f"Unsupported path character in drive id '{id}'")
 
+    flag_list = p.split(",") if len(parts) > 5 and (p := parts[5]) else []
+
     return Drive(
         id=id,
         size=parse_bytes(parts[1]),
         directory=parse_path(p) if len(parts) > 2 and (p := parts[2]) else None,
         options=p if len(parts) > 3 and (p := parts[3]) else None,
         file_id=p if len(parts) > 4 and (p := parts[4]) else id,
-        persist=parse_boolean(p) if len(parts) > 5 and (p := parts[5]) else False,
+        flags=[DriveFlag(f) for f in flag_list],
```

Comment:
This will blow up spectacularly on unknown flags. Please use `make_enum_parser(DriveFlag)(f)` or so